CMSgov/mint-app
# [EASI-4236] Change Model IT Lead and add Model Lead

# EASI-4236


## Changes and Description

- Add migration for model it lead to it lead
- renamed model it lead to it lead && add model lead
- change some translations






## How to test this change

1. Create a new discussion 
2. See if `Model Lead` and `IT Lead` are options for user roles.
3. See that `Model IT Lead` is no longer an option




## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
